filters/node.js

```diff
@@ -0,0 +1,29 @@
+import flatten from 'flat'; 
+
+export class NodeFilter {
```

Comment:
`NodeFilter` sounds like something else. It's not clear it refers to Node.js and even then there's nothing specific about Node.js here. Why not calling it `JSONFilter` or `JsonFilter`? I understand it supports some i18next-specific extensions but wouldn't it work for any arbitrary JSON file in any platform?

If you wanted to be more specific about i18next, maybe you could have a `flavor` property in the constructor to specify whether it's a "plain" or a "i18next_v4" JSON file.

Reply:
Makes sense.  I replaced node.js with json.js and called it JsonFilter